test/e2e-new/utils/util.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
I just think that we could move forward with : https://github.com/kubernetes-sigs/kubebuilder/pull/1520 
and or at least add a comment here to replace the files with when the pr get merged.

Reply:
There are comments in these files linking this PR.